monopod/pkg/commands/scaffold_test.go

```diff
@@ -0,0 +1,114 @@
+package commands
+
+import (
+	"reflect"
+	"testing"
+)
+
+const singleModuleWant = `module "apko" {
+  source            = "./images/apko"
+  target_repository = "${var.target_repository}/apko"
+}
+`
+
+const multiModuleWant = `module "apko" {
+  source            = "./images/apko"
+  target_repository = "${var.target_repository}/apko"
+}
+
+module "wolfi-base" {
+  source            = "./images/wolfi-base"
+  target_repository = "${var.target_repository}/wolfi-base"
+}
+
+module "zookeeper" {
+  source            = "./images/zookeeper"
+  target_repository = "${var.target_repository}/zookeeper"
+}
+`
+
+// this isn't the most granular test since it chains things together
+// but it does exercise the important methods and fails any invalid step
+func TestScaffoldSingleModule(t *testing.T) {
+	tests := []struct {
+		name    string
+		opts    *scaffoldOptions
+		want    *scaffoldOptions
+		wantErr bool
+	}{
+		{
+			name: "add single module to main.tf",
+			opts: &scaffoldOptions{
+				ModuleName: "apko",
+			},
+			want: &scaffoldOptions{
+				MainTf: []byte(singleModuleWant),
+			},
+			wantErr: false,
+		},
+	}
+
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
```

Comment:
Another small nit: if there's only one case, there isn't much use in the test scaffolding. You can just start with `opts := &scaffoldOptions{...}; o.addModuleToMainTf(); ...` and remove a bunch of indentation.

Reply:
Ah ok, I wasn't sure if it was sort of a default to use scaffolding. Since I can't imagine we'll be adding a bunch more test cases here, I'll simplify, thanks for the tip!